Firestore/core/test/unit/FSTGoogleTestTests.mm

```diff
@@ -82,10 +82,27 @@ @interface GoogleTests : XCTestCase
     return nil;
   }
 
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= 110000 || \
+    __MAC_OS_X_VERSION_MAX_ALLOWED >= 101300
```

Comment:
Looks like you'll need to check the iOS SDK version that the test target is being compiled against. 

https://travis-ci.org/github/firebase/firebase-ios-sdk/jobs/744997387